docs/digger/installation/macos-prereqs.asciidoc

```diff
@@ -107,24 +108,19 @@ dseditgroup -o edit -a ${USERNAME} -t user ${OSX_SSH_GROUP_NAME}
 # now, following should work  ---> ssh username@localhost
 ----
 
-This allows the `USERNAME` created previously to access the server via SSH with a password. You may also choose to add a public key to the `authorized_keys` file for said user, with the private key of the key pair existing on the Ansible management node.
-
-
-====== Connecting via Ansible
+=== Ansible root Permissions
 
 A number of tasks in the `provision-osx` Ansible role used in the installer require http://docs.ansible.com/ansible/latest/become.html[privilege escalation].
-This allows us to run commands as `sudo` user
+This allows Ansible to run commands as `sudo` user.
 
-The user created in the previous steps should be provided as a host variable to Ansible via `ansible_ssh_user`
+The name of the user created in the previous steps should be provided as a variable to Ansible in `ansible_ssh_user`.
```

Comment:
let say provided in the inventory file here.

Reply:
resolved